### PR TITLE
Solve task that calculate square roots of numbers.

### PR DESCRIPTION
**Pull request by volodymyrshkliar**

### CODE REVIEW

File: volodymyr_shkliar/squares_of_numbers.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+Calculate squares of numbers.
+"""
+
+
+gen_exp = (x ** 2 for x in range(100000000000))
```
Comment (pivancy) [inline volodymyr_shkliar/squares_of_numbers.py]: It works extremely long in Python 2.X

Comment (volodymyrshkliar) [reply volodymyr_shkliar/squares_of_numbers.py]: How can I speed up this expression or where I can find out about it?

Comment (pivancy) [reply volodymyr_shkliar/squares_of_numbers.py]: Read more about generators.

Comment (volodymyrshkliar) [reply volodymyr_shkliar/squares_of_numbers.py]: So, maybe the problem is that range in Python 2.X returns a list, and xrange returns a generator?

Comment (pivancy) [reply volodymyr_shkliar/squares_of_numbers.py]: exactly

Comment (volodymyrshkliar) [reply volodymyr_shkliar/squares_of_numbers.py]: Done.

**Comment by pivancy** [comment]
👍

**Pull request merged**